pom.xml

```diff
@@ -14,6 +14,7 @@
 
   <modules>
     <module>live-ingester</module>
+    <module>distribution</module>
```

Comment:
Here and above there shouldn't be present the `alfresco-hxinsight-connector-` prefix?